ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh

```diff
@@ -10,6 +10,9 @@ CLUSTER_NAME=${CLUSTER_NAME:-$NAMESPACE}
 CLUSTER_VERSION=${CLUSTER_VERSION:-}
 SSO_CLIENT_ID=$(cat "${CLUSTER_PROFILE_DIR}/sso-client-id")
```

Comment:
Do you need to check whether these files exist now before cat'ing them?

Reply:
Yes. I have fixed that